text_recognizer/data/iam_synthetic_paragraphs.py

```diff
@@ -40,24 +40,24 @@ def prepare_data(self, *args, **kwargs) -> None:
         rank_zero_info(
             "IAMSyntheticParagraphs.prepare_data: preparing IAM lines for synthetic IAM paragraph creation..."
         )
-        rank_zero_info("Cropping IAM line regions and loading labels...")
+
         iam = IAM()
         iam.prepare_data()
-        crops_trainval, labels_trainval = line_crops_and_labels(iam, "trainval")
-        crops_test, labels_test = line_crops_and_labels(iam, "test")
-
-        crops_trainval = [resize_image(crop, IMAGE_SCALE_FACTOR) for crop in crops_trainval]
-        crops_test = [resize_image(crop, IMAGE_SCALE_FACTOR) for crop in crops_test]
 
-        rank_zero_info(f"Saving images and labels at {PROCESSED_DATA_DIRNAME}...")
-        save_images_and_labels(crops_trainval, labels_trainval, "trainval", PROCESSED_DATA_DIRNAME)
-        save_images_and_labels(crops_test, labels_test, "test", PROCESSED_DATA_DIRNAME)
+        for split in ["train", "val", "test"]:
```

Comment:
I probably don't need to do this for `val` and `test` since we don't use these data splits

Reply:
Note to self: that's because this is for the `SyntheticParagraphs` dataset only, which doesn't have a validation or test set.

Reply:
I think we should only run this on `"train"` then. The data setup phase is one of the slower ones, and there's no good way to make it shorter right now, so it dominates e.g. a validation sanity check or a single epoch test.

I would leave it as

`for split in ["train"]:  # synthetic dataset has no validation or test data`